networking/kvm.go

```diff
@@ -420,6 +424,7 @@ func kvmTransformFlannelNetwork(net *activeNet) error {
 	net.conf.Type = n.Delegate["type"].(string)
 	net.conf.IPMasq = n.Delegate["ipMasq"].(bool)
 	net.conf.MTU = n.Delegate["mtu"].(int)
+	net.conf.IsDefaultGateway = n.Delegate["isDefaultGateway"].(bool)
```

Comment:
@jellonek What consistency you are thinking of? You mean using field
https://github.com/coreos/rkt/blob/master/networking/kvm.go#L51 ? 
It's another jSon record to cover